### PR TITLE
MSR - Improve A6/A7 damage requirements

### PR DESCRIPTION
**Pull request by dyceron**
Reduces some damage requirements in Areas 6 and 7 where it wasn't necessary. This also helps with resolver issues with low health seeds.

### CODE REVIEW

**Review by Tyranisaur — APPROVED** [review]
I was in the VC for these, this stuff looks good to me, but deserves approves from people more familiar with the game.

File: randovania/games/samus_returns/logic_database/Area 6.txt [hunk 1]
```diff
@@ -972,15 +962,12 @@ Extra - asset_id: collision_camera_060
           Any of the following:
               Space Jump or Spider Ball
               All of the following:
-                  # Reach the tunnel
+                  # Reach the Tunnel
```
Comment (MasierAzulite) [inline randovania/games/samus_returns/logic_database/Area 6.txt:965]: did you mean to change it from tunnel to Tunnel with a capital? I just happened to notice so just asking to be sure lol

Comment (dyceron) [reply randovania/games/samus_returns/logic_database/Area 6.txt:965]: Oh probably not lol. Don't think I'm gonna bother fixing it right now lol

Comment (MasierAzulite) [reply randovania/games/samus_returns/logic_database/Area 6.txt:965]: not look it changes anything yeah. other stuff seems fine to me too

Comment (dyceron) [reply randovania/games/samus_returns/logic_database/Area 6.txt:965]: Then I'll merge

**Pull request merged**